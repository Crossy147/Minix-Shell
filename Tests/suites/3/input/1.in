# basic redirections
ls servers > test.e4.1.A
ls servers/pm > test.e4.1.B
ls servers/fs > test.e4.1.C

cat test.e4.1.A > test.e4.1.C
cat test.e4.1.B >> test.e4.1.C

ls servers/rs > test.e4.1.A
cat < test.e4.1.A >> test.e4.1.D
cat < test.e4.1.B >> test.e4.1.D

sort < test.e4.1.D > test.e4.1.E

echo ----------------- test.e4.1.A
cat < test.e4.1.A
echo ----------------- test.e4.1.B
cat < test.e4.1.B
echo ----------------- test.e4.1.C
cat < test.e4.1.C
echo ----------------- test.e4.1.D
cat < test.e4.1.D
echo ----------------- test.e4.1.E
cat < test.e4.1.E

rm test.e4.1.A test.e4.1.B test.e4.1.C test.e4.1.D test.e4.1.E
