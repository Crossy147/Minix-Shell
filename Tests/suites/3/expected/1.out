----------------- test.e4.1.A
Makefile
manager.c
proto.h
rs.c
rs.h
service.c
----------------- test.e4.1.B
Makefile
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.C
Makefile
fs
init
pm
rs
Makefile
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.D
Makefile
manager.c
proto.h
rs.c
rs.h
service.c
Makefile
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.E
Makefile
Makefile
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
manager.c
misc.c
mproc.h
param.h
pm.h
proto.h
proto.h
rs.c
rs.h
service.c
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
